cubes/new_cube.py

```diff
@@ -60,8 +61,17 @@ def main(Cube):
     # Location for maximisation output
     outdir='Cube/'
 
+    #psetmins,psetmaxes,nvals=misc_functions.process_pfile(Cube[2])
+    input_dict=io.process_jfile(Cube[2])
```

Comment:
So the cube being passed lives in a json file? I have no idea what the general format of that is. The file above (H0_lite_cube.json) looks human readable - but is this adding further complexity?

Reply:
Yes, the info defining the cube resides in these JSON files.

The good and bad of JSON files is that they follow a rigid format:
Good -- you don't have to think much at all about reading and writing them
Bad -- you have to conform to them (I guess you can call that complexity)